README.md

```diff
@@ -56,8 +56,6 @@ To enable `video recording` feature you have to add the following code to the `A
 <string>Your message to user when the photo library is accessed for the first time</string>
 ```
 
-NSPhotoLibraryAddUsageDescription
```

Comment:
this permission in needed on ios

Reply:
Yes, but it's discussed immediately above, including a code sample in lines 52-57.